magefiles/utils.go

```diff
@@ -17,13 +17,41 @@ func runInPath(path string, cmd string, args ...string) error {
 	}
 
 	defer func() {
-		dirs := filepath.SplitList(path)
+		dirs := strings.Split(path, "/")
```

Comment:
@coderabbitai I don't see what is wrong here since this deferred function is call after the first os.Chdir related to the given path, the path cannot be fewer than expected. The only issue that can occurs is if path has been modified after the deferred function but it's not the case here.